src/redux/utils.js

```diff
@@ -4,7 +4,6 @@ import moment from 'moment';
 import remark from 'remark';
 import remarkReact from 'remark-react';
 import remarkEmoji from 'remark-emoji';
-import breaks from 'remark-breaks';
```

Comment:
Should we remove it from `package.json` as well? Could you run `npm rm remark-breaks` and commit lock and package files?

Reply:
Sure :)